std/array.d

```diff
@@ -3774,12 +3783,20 @@ if (isDynamicArray!A)
         return impl.capacity;
     }
 
-    /**
-     * Returns the managed array.
+    /* Use opSlice() instead.
+     * Returns: the managed array.
      */
     @property inout(ElementEncodingType!A)[] data() inout
```

Comment:
Deprecation?

Reply:
I'll leave the deprecation for the future. For starters, I simply un-documented data().